src/test/java/de/bonndan/nivio/input/SeedTest.java

```diff
@@ -47,10 +48,18 @@ public void fails() throws MalformedURLException {
 
     @Test
     public void windowsFileLocation() throws MalformedURLException {
-        Seed seed = new Seed("c:\\a\\b\\c.yml");
+        final String separator = FileSystems.getDefault().getSeparator();
+        final String seedLocation = "c:" + separator + "a" + separator + "b" + separator+ "c.yml";
+        /*
+        TODO: Should it really be file://c/a/b/c.yml? We remove the : from C: in our Seed class but if i want to open the URL without : it wont work in my browser, but dont know whats happening internally with the URL
```

Comment:
We could try to hide internals from the user, i.e. allowing anything that could be a local url, like "c:\foo\bar.yml", "./foo.yml". Given the scope of this ticket, I'd say we'd open another one to improve user experience.